modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/PlaybackManager.kt

```diff
@@ -893,10 +894,14 @@ open class PlaybackManager @Inject constructor(
                     event.message
                 }
                 val isAutomotive = Util.isAutomotive(application)
-                SentryHelper.recordException(
-                    message = "Illegal playback state encountered for episode uuid ${episode?.uuid}, isAutomotive $isAutomotive}: ",
-                    throwable = event.error ?: IllegalStateException(event.message)
-                )
+                Sentry.withScope { scope ->
+                    episode?.uuid?.let { scope.setTag("episodeUuid", it) }
+                    scope.setTag("isAutomotive", isAutomotive.toString())
```

Comment:
Do you think it would be worth having this cover wear as well with something like "appPlatform" and give it a value of either "phone", "auto", or "wear"? I'm still not completely clear, but could it be worth setting this tag at a top-level so that every event would have the "appPlatform" tag?

Reply:
Good point, I added it in 68eefa557c19a3d441ee356907c2cf64c2430f3f.

I was thinking to add Sentry for Wear app in a separate PR but I added it in this PR itself in [ea1c4a5](https://github.com/Automattic/pocket-casts-android/pull/1024/commits/ea1c4a57d546f613b36a2dde0ed4c8cc1bc80f9f).

Reply:
I tested on mobile and wear apps.

`app.platform` added as a global tag:

<img width="303" alt="mobile" src="https://github.com/Automattic/pocket-casts-android/assets/1405144/8fe6f76c-7788-46be-934a-94dbf0e31721"/>

<img width="303" alt="mobile" src="https://github.com/Automattic/pocket-casts-android/assets/1405144/aecdd3d8-3ec1-49b9-8816-ab030d200407"/>